### PR TITLE
all: use wasm-tools-go@v0.2.0

### PR DESCRIPTION
**Pull request by ydnar**
`wasm-tools-go` moved to the @bytecodealliance org: https://github.com/bytecodealliance/wasm-tools-go

### CODE REVIEW

File: internal/wasm-tools/go.mod [hunk 1]
```diff
@@ -2,11 +2,11 @@ module github.com/tinygo-org/tinygo/internal/tools
 
 go 1.22.4
 
-require github.com/ydnar/wasm-tools-go v0.1.4
+require github.com/ydnar/wasm-tools-go v0.1.5
```
Comment (dgryski) [inline internal/wasm-tools/go.mod]: Is this still correct?

**Review by dgryski — APPROVED** [review]
Approved unless the `go.mod` line is wrong..

File: internal/wasm-tools/go.mod (hunk 1 above)
Comment (ydnar) [reply internal/wasm-tools/go.mod]: No. Thanks!

**Review by dgryski — APPROVED** [review]
LGTM

**Comment by deadprogram** [comment]
Thank you for updating @ydnar and to @dgryski for review. Now merging.

**Pull request merged**